### DISCUSSION TITLE
how to handle this problem!!

### DISCUSSION THREAD

**Discussion by zhangwei199207** [crawlab-team/crawlab#1405] [Q&A] [answered]
![image](https://github.com/crawlab-team/crawlab/assets/72115258/24d3a709-c103-49a6-88b2-25d94dab1147)

**Comment by zhangwei199207** [comment]
some button disappeard

**Reply by tikazyq** [reply]
Can you elaborate a bit more? This seems okay

**Reply by zhangwei199207** [reply]
I deployed on centos7,here is my env:Docker version 24.0.6,docker-compose version 1.18.0.But got some button lost in pages.
![image](https://github.com/crawlab-team/crawlab/assets/72115258/451a3b66-d853-42ea-bb1c-5292d4ff5aea)

**Reply by zhangwei199207** [reply]
this is the log,and how to solve
[192.168.13.9-1697180483320.log](https://github.com/crawlab-team/crawlab/files/12890521/192.168.13.9-1697180483320.log)

**Reply by tikazyq** [reply, marked as answer]
Which version are you using?

**Reply by zhangwei199207** [reply]
Thankyou,master Zhang,0.6.3 is the perfect version !!!

**Comment by zhangwei199207** [comment]
yeah,I find this too,I pull the latest，it's 0.6.0，but on windows and Ubuntu,it's 0.6.3,thanks



---Original---
From: "Marvin ***@***.***&gt;
Date: Fri, Oct 13, 2023 17:33 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [crawlab-team/crawlab] how to handle this problem!! (Discussion#1405)




 
Which version are you using?
